docs/development-and-testing.md

```diff
@@ -1,34 +1,56 @@
 ## Development and Testing
-0. You would need to have `quay.io` account, running docker and kubernetes; and make sure `quay.io/<QUAY_USERNAME>/direct-csi` repository is `public`.
-1. Go to project root
+1. You would need to have `quay.io` account, running docker and kubernetes.
+2. Docker login to your `quay.io` account.
+```bash
+$ docker login --username <QUAY_USERNAME> https://quay.io
+```
+3. If you do not have `csi-provisioner`, `livenessprobe` or `csi-node-driver-registrar`, create those repositories by pull/tag/push to your `quay.io` account.
```

Comment:
This operation is idempotent. We should just say

Creates the following images in your registry account:

1. ..
2. ..
3. ..

Reply:
Fixed in different way.